draft-ietf-quic-qlog-h3-events.md

```diff
@@ -333,11 +307,10 @@ HTTPFrameParsed = {
 ~~~
 {: #http-frameparsed-def title="HTTPFrameParsed definition"}
 
-Note: in HTTP/3, DATA frames can have arbitrarily large lengths to reduce frame
-header overhead. As such, DATA frames can span many QUIC packets and can be
-processed in a streaming fashion. In this case, the frame_parsed event is emitted
-once for the frame header, and further streamed data is indicated using the
-data_moved event.
+HTTP/3 DATA frames can have arbitrarily large lengths to reduce frame header
+overhead. As such, DATA frames can span multiple QUIC packets. In this case, the
+frame_parsed event is emitted once for the frame header, and further streamed
+data is indicated using the data_moved event.
```

Comment:
This seems like it can be genericised and moved somewhere else: there is no tight coupling between H3 and QUIC frame boundaries meaning a H3 frames could span many packets, or many frames could be in a single packet.

Implementations can deploy streaming strategies to any H3 frame parsing. I agree that DATA frames are the most likely, but really any frame could be larger than a packet. 

Furthermore, due to varint H3 frame fields and the lack of preserved boundaries, some implementations might generate data_moved events before frame_parsed events. Cloudlfare quiche does this, for example. 

The text is better than the old text, so lets not block the PR. I'll open a separate issue.